### DISCUSSION TITLE
点击保存怎么获取json-schema配置？怎么把从服务端读取的配置在渲染出来？

### DISCUSSION THREAD

**Discussion by liuboshuo** [alibaba/designable#16] [Q&A]
大佬：
1、点击保存怎么获取json-schema配置？
2、怎么把从服务端读取的配置在渲染出来？

**Comment by janryWang** [comment]
等文档吧，没文档前只能自己看源码了

**Reply by liuboshuo** [reply]
文档有计划什么时候放出来吗？

**Reply by janryWang** [reply]
9月份吧

**Reply by sstruct** [reply]
11月了😄

**Reply by lidianhao123** [reply]
文档有什么计划吗？有参与的机会不？

**Comment by giligiliduang** [comment]
在工程目录下 formily/transformer/src/index.ts，提供了一个transformToSchema方法，应该是把组件树的根节点传进去就能拿到转换好的json-schema了
